### PR TITLE
fix bandwidth typo in RocketCDN tab

### LINKED ISSUE #5353: RocketCDN tab - Misspelled word bandwidth
**Before submitting an issue please check that you’ve completed the following steps:**
- Made sure you’re on the latest version
- Used the search feature to ensure that the bug hasn’t been reported before

**Describe the bug**
In WP Rocket settings, on the CDN tab, the word bandwidth is misspelled. The word "bandwith" should be corrected to "bandwidth".
This text string is [located here](https://github.com/wp-media/wp-rocket/blob/dafdbedd6d30bb7d2eb74e622f178e100a723a47/inc/Engine/CDN/RocketCDN/views/cta-big.php#L42).

**To Reproduce**
1. Go to WP Rocket settings
2. Click on CDN tab
3. If you don't see RocketCDN banner with information about subscription, click on the **Learn More** button

**Expected behavior**
Use correctly spelled word

**Screenshots**
https://i.imgur.com/kRtyrfy.jpeg

**Additional context**

**Backlog Grooming (for WP Media dev team use only)**
- [ ] Reproduce the problem
- [ ] Identify the root cause
- [ ] Scope a solution
- [ ] Estimate the effort

### PR DESCRIPTION
**Pull request by mostafa-hisham**
## Description

fix bandwidth typo

Fixes #5353 

## Type of change 

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (non-breaking change which improves an existing functionality)

## Is the solution different from the one proposed during the grooming?

No
## How Has This Been Tested?

- [x] Tested locally

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by jeawhanlee — APPROVED** [review]

**Comment by Mai-Saad** [comment]
The typo is fixed now but the translation is not displayed as it was on the trunk, @engahmeds3ed  do we need further changes on PR so translation works?

File: tests/Fixtures/inc/Engine/CDN/RocketCDN/NoticesSubscriber/displayRocketcdnCta.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		<p class="wpr-rocketcdn-cta-subtitle">Speed up your website thanks to:</p>
 		<div class="wpr-flex">
 			<ul class="wpr-rocketcdn-features">
-				<li class="wpr-rocketcdn-feature wpr-rocketcdn-bandwidth">High performance Content Delivery Network (CDN) with <strong>unlimited bandwith</strong></li>
+				<li class="wpr-rocketcdn-feature wpr-rocketcdn-bandwidth">High performance Content Delivery Network (CDN) with <strong>unlimited bandwidth</strong></li>
```
Comment (engahmeds3ed) [inline tests/Fixtures/inc/Engine/CDN/RocketCDN/NoticesSubscriber/displayRocketcdnCta.php:28]: Why those strings are not translatable?
@mostafa-hisham Do u think it's the time to change it to use the WordPress translation functions?

**Review by engahmeds3ed — CHANGES_REQUESTED** [review]

File: tests/Fixtures/inc/Engine/CDN/RocketCDN/NoticesSubscriber/displayRocketcdnCta.php (hunk 1 above)
Comment (engahmeds3ed) [reply tests/Fixtures/inc/Engine/CDN/RocketCDN/NoticesSubscriber/displayRocketcdnCta.php:28]: those are tests, sorry I didn't notice that :D

**Review by Mai-Saad — APPROVED** [review]
As per discussion with @engahmeds3ed nothing is needed on PR

**Pull request merged**